server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/webhook/WebhookPostTask.java

```diff
@@ -48,16 +48,23 @@ public WebhookPostTask(ConfigurationRepository configRepository, WebhookPayloadF
   public void finished(ProjectAnalysis analysis) {
     Configuration config = configRepository.getConfiguration();
 
-    webHooks.sendProjectAnalysisUpdate(
-      config,
-      new WebHooks.Analysis(analysis.getProject().getUuid(), analysis.getCeTask().getId()),
-      () -> payloadFactory.create(convert(analysis)));
+    // FIXME we should not rely on this test
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)